src/mqtt/transport/quic/mqtt_quic.c

```diff
@@ -175,12 +176,12 @@ mqtt_quic_pipe_timer_cb(void *arg)
 	nni_mtx_lock(&p->mtx);
 	if (!p->busy && !nni_aio_busy(p->qsaio)) {
 		// send pingreq
-		buf[0] = 0xC0;
-		buf[1] = 0x00;
+		pingbuf[0] = 0xC0;
+		pingbuf[1] = 0x00;
```

Comment:
if pingbuf always equals {0xC0, 0x00}, Maybe we can just initialize the variables in the first place?

Reply:
correct